pv211_utils/datasets.py

```diff
@@ -0,0 +1,724 @@
+"""The dataset module provides interfaces for loading
```

Comment:
Use classes from this modules in the following Jupyter notebooks and rerun them to see if they are working:

- `notebooks/cranfield.ipynb`
- `notebooks/trec.ipynb`
- `notebooks/arqmath.ipynb`

For `notebooks/trec.ipynb` and `notebooks/arqmath.ipynb`, the free runtime provided by Google Colab will be insufficient. For these, ise [the PV211 JupyterHub][1] instead.

 [1]: https://iirhub.cloud.e-infra.cz/

Reply:
todo

Reply:
Not necessarily a priority for the second term project, but we will want this sooner or later. There should be no dead code. All functionality except the most experimental bits should be exposed in the notebooks, so that students can use them in their projects.